libavcodec/qsvenc.c

```diff
@@ -644,6 +644,12 @@ static int check_enc_param(AVCodecContext *avctx, QSVEncContext *q)
     return 1;
 }
 
+static int is_adaptive_gop(QSVEncContext *q) {
+    if (q->adaptive_b || q->adaptive_i || q->b_strategy || q->p_strategy)
```

Comment:
The default value for adaptive_b / adaptive_i is -1

Reply:
Oh, thanks. I will update it.